### DISCUSSION TITLE
[ERROR] PostgreSQL LXC install.

### DISCUSSION THREAD

**Discussion by ianalexis** [tteck/Proxmox#1793] [Help Needed] [answered]
PVE: 7.4-16
LXC Os: Bebian 11 and Ubuntu 22
[ERROR] in line 33: exit code 1: while executing command cat <<EOF > /etc/postgresql/15/main/pg_hba.conf

**Comment by tteck** [comment, marked as answer]
[Fixed](https://github.com/tteck/Proxmox/commit/6df6d1c2fb27c5480e64e7aba8908b4cae12eafe)
